interpreter/text/parser.mly

```diff
@@ -641,6 +651,11 @@ func_body :
     { fun c -> ignore (bind_local c $3); let f = $6 c in
       {f with locals = $4 :: f.locals} }
 
+/* Exceptions */
```

Comment:
Nit: extra new line after; or better, move this into Tables, mems & globals section below